src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,585 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
+import { DropdownButton, DropdownItem, Dropdown } from 'react-bootstrap';
+
+import { JWT, console, object, layout, ajax, navigate } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
+import { Alerts } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/Alerts';
+
+import { ItemFileAttachment } from './components/ItemFileAttachment';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ItemFileAttachment' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)